pynecone/.templates/jinja/web/pages/base_page.js.jinja2

```diff
@@ -4,6 +4,7 @@
 {% for module in imports%}
   {{- utils.get_import(module) }}
 {% endfor %}
+import env from  "{{env_file}}"
```

Comment:
I think we can get rid of this - the constants should only be needed in the `state.js` file now